setup.py

```diff
@@ -24,7 +24,7 @@
     author='hipo',
     author_email='pypi@hipolabs.com',
     url='https://github.com/Hipo/drf-extra-fields',
-    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
+    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*",
```

Comment:
What about `>=2.7, >=3.5`?

Reply:
Sounds good to me. FYI: @omerfarukabaci

Reply:
Fixed accordingly, thank you! 👍

Reply:
According to the documentation of **Python Packaging Authority (PyPA)**, unused versions should be explicitly written with `!={version}`. You can find the related documentation below.

https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires